### PR TITLE
Feat: Add ADD ALL CHAPTER PAGE ADMIN

### PR DESCRIPTION
**Pull request by daveclinton**
# Feat: Add ADD ALL CHAPTER PAGE
This PR holds a change that adds the all chapters ad page 
image
image
image

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Tested manually on the browser

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by JimmyTron — COMMENTED** [review]
The implement is ok but i only have one issue 
the next button allows me to move even with no data input on the form which could potentially make errors the social links maybe ok but not on the general info

File: src/ADMIN/components/Organizers.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          <button
+            type="button"
+            className="w-[154px] h-11 px-3.5 py-3 rounded-lg border border-emerald-600 text-emerald-600 text-[13px] font-medium leading-tight tracking-tight justify-end items-center gap-2.5 inline-flex"
+          >
+            <FontAwesomeIcon
+              icon={faPlus}
+              size="xl"
+              style={{ color: "#009975" }}
+            />
+            Add Organizer
```
Comment (Rioba-Ian) [inline src/ADMIN/components/Organizers.jsx:87]: We can add a click handler here to add another section for adding another organizer with the form details just as above

**Comment by daveclinton** [comment]
> The implement is ok but i only have one issue the next button allows me to move even with no data input on the form which could potentially make errors the social links maybe ok but not on the general info

@Rioba-Ian I think that should be done together with the integration and when i have a full user story for the functionalitu, here I was just given the design. Please advice.

File: src/ADMIN/components/Organizers.jsx (hunk 1 above)
Comment (daveclinton) [reply src/ADMIN/components/Organizers.jsx:87]: @Rioba-Ian I was fully focusing on implementing the design and the stepper movement. I wasn't given a user story for this section. Also, adding an organizer should be sent to some endpoint i think. If that is ready I can work on the integration asap.

Comment (JimmyTron) [reply src/ADMIN/components/Organizers.jsx:87]: http://34.175.147.171:8000/api/chapter/

Comment (JimmyTron) [reply src/ADMIN/components/Organizers.jsx:87]: http://34.175.147.171:8000/#tag/chapter

Comment (Rioba-Ian) [reply src/ADMIN/components/Organizers.jsx:87]: @daveclinton , I thought the add organizer added another form with the inputs. Then it's alright as it is.

**Review by Rioba-Ian — APPROVED** [review]

File: src/ADMIN/components/StepIndicator.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                  <FontAwesomeIcon
+                    icon={faCheck}
+                    style={{ color: "#fafafa" }}
+                  />
+                ) : (
+                  step.section
+                )}
+              </div>
+            </div>
+            <div className="flex-col justify-center items-start inline-flex">
```
Comment (sonylomo) [inline src/ADMIN/components/StepIndicator.jsx:42]:
```suggestion
            <div className="flex-col justify-center items-start hidden sm:inline-flex">
```

Comment (sonylomo) [reply src/ADMIN/components/StepIndicator.jsx:42]: Just a tiny nitpick: my suggestion will hide the `GENERAL INFO`, `SOCIAL MEDIA` and `ORGANIZERS` labels on mobile view.

**Pull request merged**